renderer/RenderSystem_init.cpp

```diff
@@ -234,14 +235,11 @@ idCVar r_glCoreProfile( "r_glCoreProfile", "2", CVAR_RENDERER | CVAR_ARCHIVE,
 idCVarBool r_newFrob( "r_newFrob", "0", CVAR_RENDERER | CVAR_ARCHIVE, "1 = use the frob shader instead of material stages" );
 
 // FBO
-idCVar r_useFbo( "r_useFBO", "1", CVAR_RENDERER | CVAR_BOOL, "Use framebuffer objects" );
 idCVar r_showFBO( "r_showFBO", "0", CVAR_RENDERER | CVAR_INTEGER, "0-5 individual fbo attachments" );
-idCVar r_fboColorBits( "r_fboColorBits", "32", CVAR_RENDERER | CVAR_INTEGER | CVAR_ARCHIVE, "15, 32, 64" );
+idCVar r_fboColorBits( "r_fboColorBits", "64", CVAR_RENDERER | CVAR_INTEGER | CVAR_ARCHIVE, "32, 64" );
```

Comment:
Already a default? Huh...

Reply:
Yes. I mean, this is targeted at 2.09, which, by our current release intervals, is quite some time in the future. In that period I think a reasonable amount of people will have used 64 bits with 2.08, and I also plan to do some initial alpha testing with a few people for the FBO and draw call reduction changes to hopefully catch the most glaring issues early. If in that time or in the beta period for 2.09 we do find some major issues with 64 bits, we can revert to 32. But from what I've seen, the performance impact is minimal, and the banding reduction and potential for bloom is significant enough that I think we should aim for 2.09 to default to 64 bits, if at all possible :)